content/_data/sidebar.json

```diff
@@ -12,6 +12,10 @@
               "title": "Why Cypress?",
               "slug": "why-cypress"
             },
+            {
+              "title": "Choosing E2E or Component Tests",
```

Comment:
Should we call this "Choosing a Testing Type" with a slug like `choosing-testing-type`?

(Eventually, we're going to have Node tests)